x-pack/plugins/spaces/server/lib/create_default_space.test.ts

```diff
@@ -79,17 +81,25 @@ const createMockServer = (settings: MockServerSettings = {}) => {
     return settings[key];
   });
 
-  return mockServer;
+  return {
+    config: mockServer.config(),
+    savedObjects: (mockServer.savedObjects as unknown) as SavedObjectsService,
+    elasticsearch: ({
+      dataClient$: Rx.of({
```

Comment:
TODO for myself: core should provide dataClients with mocked `ClusterClient` interface